addons/ccm-kubevirt/config-map.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloud-config
+  namespace: kube-system
+data:
+  cloud-config: |
+    kubeconfig: /var/run/secrets/infracluster/KUBEVIRT_KUBECONFIG
+    namespace: {{ .Config.CloudProvider.KubeVirt.InfraNamespace }}
+    instancesV2:
+      zoneAndRegionEnabled: {{ .Config.CloudProvider.KubeVirt.ZoneAndRegionEnabled }}
+    loadBalancer:
+      enabled: {{ .Config.CloudProvider.KubeVirt.CCMLoadBalancerEnabled }}
```

Comment:
This is a bit "NPE-unsafe" code, I'm not sure if we can trigger NPE somehow, but I think it's not a bad idea to guard it just in case, who knows.

Reply:
:heavy_check_mark: 
also added validation for required fields